mix.exs

```diff
@@ -33,6 +33,7 @@ defmodule Sparrow.MixProject do
       {:joken, "~> 2.0-rc0"},
       {:poison, "~> 3.1"},
       {:mock, "~> 0.3.0", only: :test},
+      {:meck, github: "eproxus/meck", override: true},
```

Comment:
Do you need Erlang `:meck` since there already is Elixir `:mock` included?

Reply:
Without `:meck` included there were problems with compatibility with some OTP versions AFAIR